AppServices/Validations/UpdateCustomerValidator.cs

```diff
@@ -0,0 +1,115 @@
+﻿using APIDesafioWarren.Models;
+using Application.Models.DTOs;
+using FluentValidation;
+using FluentValidation.Validators;
+using System;
+
+namespace AppServices.Validations
+{
+    public class UpdateCustomerValidator : AbstractValidator<UpdateCustomerRequest>
+    {
+        public UpdateCustomerValidator()
+        {
+            RuleFor(c => c.FullName)
+                .NotEmpty()
+                .Must(v => v.ValidFullName())
+                .MaximumLength(300)
+                .MinimumLength(6);
+
+            RuleFor(c => c.Email)
+                .NotEmpty()
+                .MinimumLength(6)
+                .MaximumLength(254)
+                .EmailAddress(EmailValidationMode.Net4xRegex);
+
+            RuleFor(c => c.Cpf)
+                .NotEmpty()
+                .Must(ValidCPF)
+                .MinimumLength(11)
+                .MaximumLength(14);
+
+            RuleFor(c => c.Country)
+                .NotEmpty()
+                .Must(v => v.AllCharacteresArentEqualsToTheFirstCharacter())
```

Comment:
```suggestion
                .Must(v => v.IsValidString)
```
Esse método já contém a validação anterior e mais outras validações